rero_ils/modules/holdings/jsonschemas/holdings/holding-v0.0.1.json

```diff
@@ -590,6 +609,252 @@
           }
         }
       }
+    },
+    "acquisition_status": {
+      "title": "Acquisition status",
+      "description": "Indicates whether newly published parts of a multipart or serial item are being received.",
+      "type": "string",
+      "enum": [
+        "unknown",
+        "other_receipt_or_acquisition_status",
+        "received_and_complete_or_ceased",
+        "on_order",
+        "currently_received",
+        "not_currently_received"
```

Comment:
@pronguen What is the advantage to have `marc_acquisition_status:0` instead of `on_order` (for example)? I'm not sure, but it's easier to code with name that mean something.

Reply:
I share the same opinion of @iGormilhit .

Reply:
> it's easier to code with name that mean something

1. Is there really a difficulty? In the form options, you have for each enum: label + value identical...
2. It is a matter of semantic: we use the MARC content for this field, so keeping the MARC code have the following advantages:
  * the documentation of the field is available on the MARC pages
  * an export of our data to MARC is then easier, as the MARC code are present
  * the code in the form of `marc_acquisition_status:5` is transparent and identical to a URI in the semantic web... After all, it is like using a controlled vocabulary.

This is just to take a decision. I am open whatever the choice is. But my fear is to copy the MARC structure in our JSON and then to loose the history.

Reply:
From my point of view `marc_acquisition_status:5` has no meaning. It is required to learn Marc to understand the format. It would be great to have bibframe standard value for this. If it does not exists I would prefer to have our own label: i.e. `on_order`. Moreover if I have to do an expert search I prefer: `holding_status:on_order` instead of `holding_status:marc_acquisition_status:5` which can cause some troubles. This is just my opinion.

Reply:
Ok, I asked the opinion of @ThierryClavel. I observed a difficulty with having a label in natural language: for some label, we need an precise translation and knowing that it comes from MARC helps to translate it.
In this example (https://hosted.weblate.org/translate/rero_plus/sonar/en/?checksum=8bb223dcada1eb5a&q), we know that it is a role in the contribution field of the document, and that it should be translated in "editor" and not in "publisher".
I

Reply:
Ok, after discussion, we stay with the human readable codes without prefix.
Guidelines for enumerations in JSON schema
* use URI if available
* use prefixes from 8-10 values
* avoid semi-colon in the values